content/source/docs/cloud/sentinel/import/tfconfig-v2.html.md

```diff
@@ -0,0 +1,370 @@
+import can access all aspects of the configuration: providers, resources,
+data sources, modules, and variables.
+
+Some use cases for `tfconfig` include:
+
+* **Organizational naming conventions**: requiring that configuration elements
+  are named in a way that conforms to some organization-wide standard.
+* **Required inputs and outputs**: organizations may require a particular set
+  of input variable names across all workspaces or may require a particular
+  set of outputs for asset management purposes.
+* **Enforcing particular modules**: organizations may provide a number of
+  "building block" modules and require that each workspace be built only from
+  combinations of these modules.
+* **Enforcing particular providers or resources**: an organization may wish to
+  require or prevent the use of providers and/or resources so that configuration
+  authors cannot use alternative approaches to work around policy
+  restrictions.
+
+## Import Overview
+
+The `tfconfig/v2` import is structured as a series of _collections_, keyed as a
+specific format, such as resource address, module address, or a
+specifically-formatted provider key.
+
+The collections are:
+
+* [`providers`](#the-providers-collection) - The configuration for all provider
+  instances across all modules in the configuration.
+* [`resources`](#the-resources-collection) - The configuration of all resources
+  across all modules in the configuration.
+* [`variables`](#the-variables-collection) - The configuration of all variable
+  definitions across all modules in the configuration
+* [`outputs`](#the-outputs-collection) - The configuration of all output
+  definitions across all modules in the configuration.
+* [`module_calls`](#the-variables-collection) - The configuration of all module
+  calls (individual [`module`](/docs/configuration/modules.html) blocks) across
+  all modules in the configuration.
+
+These collections are specifically designed to be used with the
+[`filter`](https://docs.hashicorp.com/sentinel/language/collection-operations/#filter-expression)
+quantifier expression in Sentinel, so that one can collect a list of resources
+to perform policy checks on without having to write complex module or
+configuration traversal. As an example, the following code will return all
+`aws_instance` resource types within the configuration, regardless of what
+module they are in:
+
+```
+all_aws_instances = filter tfconfig.resources as _, r {
+	r.mode is "managed" and
+		r.type is "aws_instance"
+}
+```
+
+You can add specific attributes to the filter to narrow the search, such as the
+module address. The following code would return resources in a module named
+`foo` only:
+
+```
+all_aws_instances = filter tfconfig.resources as _, r {
+	r.module_address is "module.foo" and
+		r.mode is "managed" and
+		r.type is "aws_instance"
+}
+```
+
+### "Approximate" Addresses Explained
+
+Throughout this documentation, the term _approxiamte address_ will be used. This
```

Comment:
Typo on `approximate` here

Reply:
Fixed! Looks like my markdown highlighting in vim likes to hide spelling error highlights when using italics. :P